internal/parser/planparserv2/parser_visitor.go

```diff
@@ -1370,3 +1370,213 @@ func (v *ParserVisitor) VisitArrayLength(ctx *parser.ArrayLengthContext) interfa
 		nodeDependent: true,
 	}
 }
+
+func (v *ParserVisitor) VisitGeospatialEuqals(ctx *parser.GeospatialEuqalsContext) interface{} {
+	columnInfo, err := v.getChildColumnInfo(ctx.Identifier(), ctx.StringLiteral())
+	if err != nil {
+		return err
+	}
+	if columnInfo == nil ||
+		(!typeutil.IsGeospatialType(columnInfo.GetDataType())) {
+		return fmt.Errorf(
+			"Euqals operation are only supported on geospatial fields now, got: %s", ctx.GetText())
+	}
+	element := ctx.GetChild(4).(antlr.ParseTree).GetText() // the wkt input
```

Comment:
elements, ok := ctx.GetChild(4).(antlr.ParseTree).GetText()
add `ok` to ensure it won't panic

Reply:
and why get `4` for ctx child? what's the meaning?